frontend/src/v5/store/containers/containers.selectors.ts

```diff
@@ -79,7 +79,7 @@ export const selectContainerUsers = createSelector(
 	selectCurrentTeamspaceUsersByIds,
 	(container, users) => {
 		if (isEmpty(users)) return [];
-		return (container?.users || []).map((user) => ({ ...user, ...users[user.user] }));
+		return (container?.users || []).map((user) => ({ ...user, ...users[user.user], isNotTeamspaceUser: !some(users, ((u) => u.user === user.user)) }));
```

Comment:
I dont think is necesarry to use "some" users is a dictionary
```suggestion
				return (container?.users || []).map((user) => ({ ...user, ...users[user.user], isNotTeamspaceUser: !Boolean(users[user.user])}));
```